fp.js

```diff
@@ -6,9 +6,10 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-people.forEach(function(person){
-  peoplesAges.push(person.age)
-});
+// people.forEach(function(person){
+//   peoplesAges.push(person.age)
+// });
+  peoplesAges = people.map((x,i)=>people[i].age);
 
```

Comment:
`x` is equal to `people[i]` in your example. This is correct although stylistically this is preferable: 

```js
peoplesAges = people.map( person => person.age);
```

I would avoid using non-semantic variables like `x` whenever possible.